patterns/pattern-1/analytics-mod-stdln-srv.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2ei-analytics-mod-stdln-srv
+spec:
+  type: NodePort
+  selector:
+    dpl: wso2ei-analytics-mod-stdln
+  ports:
+  - name: 'servlet-http'
+    port: 9764
+    targetPort: 9764
+    nodePort: 31031
```

Comment:
When static node ports are used, if they have been already used in the given K8S environment the deployment would fail. Therefore, it might be better to use dynamic node ports for exposing ports to external networks. May be an internal or external load balancer can be used for this purpose.